README.md

```diff
@@ -947,6 +947,35 @@ const bankAccount = new BankAccount();
 // Buy shoes...
 bankAccount.withdraw(100);
 ```
+
+**Good with getter/setter**:
+```javascript
+class BankAccount {
+  constructor(balance = 1000) {
+    this_balance = balance;
+  }
+
+  // It doesn't have to be prefixed with `get` or `set` to be a getter/setter
+  set balance(amount) {
+    if (verifyAmountCanBeSetted(amount)) {
```

Comment:
While you're at it, can you write a fake method for `verifyAmountCanBeSettled`? It doesn't have to have any code other than `// ...` Thanks!